dnf/cli/cli.py

```diff
@@ -985,6 +985,12 @@ def _read_conf_file(self, releasever=None):
         # search config file inside the installroot first
         conf._search_inside_installroot('config_file_path')
 
+        # check, that config file set in CLI, exists
```

Comment:
What about `check whatever a config file is requested from a commandline and the file exists`?